splinter

```diff
@@ -46,11 +46,16 @@ main() {
   local i
   for i in "${!regexes[@]}"; do
     local matches
-    matches="$(grep -EHInr "${regexes[$i]}" "${src_paths[@]}" || echo)"
+    local should_ignore
+    matches="$(grep -EHInr "${regexes[$i]}" "$PWD/${src_paths[@]}" || echo)"
```

Comment:
I forgot about this PWD thing. In my version, it had `$(PWD)` but was failing sometimes. Looks like it was gone here in the upstream?